decidim-sortitions/app/views/decidim/sortitions/admin/sortitions/edit.html.erb

```diff
@@ -1,32 +1,30 @@
 <% add_decidim_page_title(translated_attribute(sortition.title)) %>
 <div class="item_show__header">
   <h2 class="item_show__header-title">
-    <%= decidim_html_escape(translated_attribute(sortition.title)) %>
+    <%= t ".title" %>
   </h2>
 </div>
-<div class="item__edit item__edit-1col">
+<div>
```

Comment:
Let's leave the 1col design, as its a bit better in large screens and more consistent with the rest of the UI 

```suggestion
<div class="item__edit item__edit-1col">
```